pkg/lib/page.scss

```diff
@@ -1,6 +1,7 @@
 @use "@patternfly/patternfly/base/patternfly-themes.scss";
 @use "@patternfly/patternfly/patternfly-theme-dark.scss";
 @use "./patternfly/patternfly-4-overrides.scss";
+@use "./patternfly/patternfly-5-overrides.scss";
```

Comment:
Is this a temporary thing? It smells dangerous to include both overrides in all pages. They both use `.pf-` prefixes (not `.pf5` as I initially thought), so they could collide. Should we move the override include into each page individually rather? And let each page pick if it's 4 or 5?

Reply:
Actually it's not possible to pick if it's 4 or 5. All cockpit repo will use PF5 after this PR. The patternfly-4-overrides that are still relevant means that the bugs made it also to the PF5. I will merge the two files and name it patternfly-5-overrides.scss.